pkg/controllers/disruption/consolidation_test.go

```diff
@@ -227,6 +229,9 @@ var _ = Describe("Consolidation", func() {
 			ExpectReconcileSucceeded(ctx, disruptionController, client.ObjectKey{})
 			wg.Wait()
 
+			// Expect the cluster consolidated state to not change.
+			Expect(disruptionController.Methods[3].(*disruption.EmptyNodeConsolidation).IsConsolidated()).To(BeFalse())
```

Comment:
This feels extremely dirty to me with exposing something publicly, getting a known ordering of it and then casting to get access to a member function. Is there a cleaner way to handle this that doesn't cause this many different hoops to jump through to get the testing to work-out?